app/services/open_library_service.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class OpenLibraryService
+  include HTTParty.freeze
```

Comment:
lol the .freeze is for the constant -> https://blog.honeybadger.io/when-to-use-freeze-and-frozen-in-ruby/